process/sync/baseSync.go

```diff
@@ -666,10 +666,9 @@ func (boot *baseBootstrap) rollBackOneBlock(
 
 		// TODO check if pruning should be done on rollback
 		if boot.accounts.IsPruningEnabled() {
```

Comment:
This whole block can be put into a function

Reply:
Done.